scripts/dockerfiles/Dockerfile.release

```diff
@@ -20,6 +20,9 @@ COPY ["LICENSE", "NOTICE", "/"]
 
 COPY out/amazon-ecs-pause.tar /images/amazon-ecs-pause.tar
 
+# Copy our cni plugins ecs-eni, ecs-ipam and ecs-bridge
+COPY out/cni-plugins /cni-plugins
```

Comment:
If you want to use this value for the default path, can you also modify the value [here](https://github.com/aws/amazon-ecs-agent/blob/enis/agent/config/config.go#L85) and the README file?